dolphinscheduler-ui/src/locales/en_US/user-dropdown.ts

```diff
@@ -18,5 +18,6 @@
 export default {
   profile: 'Profile',
   password: 'Password',
+  product: 'Product',
```

Comment:
I would prefer to use `about` instead of `Product`.

Reply:
> I would prefer to use `about` instead of `Product`.

These issues mentioned has been resolved